src/dnvm/InstallCommand.cs

```diff
@@ -50,14 +50,22 @@ public static Task<Result> Run(DnvmEnv env, Logger logger, DnvmSubCommand.Instal
             targetDir = (dirPath, env.CwdFs);
         }
 
-        return Run(env, logger, new Options
+        Result result = await Run(env, logger, new Options
         {
             SdkVersion = args.SdkVersion,
             Force = args.Force ?? false,
             SdkDir = args.SdkDir,
             Verbose = args.Verbose ?? false,
             TargetDir = targetDir,
         });
+
+        if (result is not Result.Success && args.SdkVersion.Patch < 100)
+        {
+            env.Console.Error($"Requested SDK version '{args.SdkVersion}' with patch '{args.SdkVersion.Patch}'. " +
+            "The dotnet SDK typically has 3 digit patches versions. " +
+            $"Did you mean '{args.SdkVersion.WithSuggestedThreeDigitPatch()}'?");
```

Comment:
Idea is good, but unfortunately I think it's pretty common for the runtime patch version and the SDK version to skew pretty far. That is, I think it's very unlikely that SDK == patch * 100. For instance, runtime 9.0.9 actually corresponds to SDK 9.0.305 and 9.0.110.

I think we should just spit the warning that the version is likely incorrect and let the user find the correct corresponding SDK.